src/main/java/picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.java

```diff
@@ -37,7 +37,7 @@
        - int: read1ReferenceIndex, read1Coordinate, read2ReferenceIndex, read2Coordinate, duplicateSetSize
        - long: read1IndexInFile, read2IndexInFile
      */
-    protected static final int SIZE_OF = (1 * 1) + (5 * 2) + (5 * 4) + (8 * 2) + 1
+    protected static final int SIZE_OF = (1 * 1) + (5 * 2) + (5 * 4) + (8 * 2) + 1 + 1
```

Comment:
Explain the new extra byte (from firstOfPair) in the comment above.

Reply:
@als364, thanks for the review!
Please have a look at my comment about `SIZE_OF` value: https://github.com/broadinstitute/picard/pull/1035#discussion_r165039319.

Reply:
That comment makes sense, thanks!